static/js/components/analytics/AnalyticsChart.js

```diff
@@ -135,14 +146,12 @@ export class AnalyticsChart extends React.Component {
           dependentAxis
           label="views"
           offsetX={padding.left - 2}
+          axisLabelComponent={
+            <VictoryLabel dy={-1 * (baseAxisLabelStyle.fontSize + 5)} />
```

Comment:
Is `fontSize` a pixel value here?

Reply:
this `fontSize` is a value in terms of SVG units